docs/vocascan-server/run-programmatically.md

```diff
@@ -0,0 +1,50 @@
+# Run programmatically
+
+We also provide a way to integrate the vocascan-server directly into your nodeJS application. This could be useful if
+you want to run vocascan-server from inside of your electron client.
```

Comment:
`This could be useful if
you want to run vocascan-server within of your electron client.` sounds better?